### PR TITLE
[DOCS-7133] Add specifying library config for Host and Docker.

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

Config to specify languages and library versions in single step instrumentation is being added to Host and Docker.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by buraizu — APPROVED** [review]
Approving with some non-blocking suggestions

File: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md [hunk 1]
```diff
@@ -75,11 +104,45 @@ For a Docker Linux container:
...
+      <ul>
+         <li><a href="#lib-docker">Specifying tracing library versions.</a></li>
+         <li><a href="#env-docker">Tagging observability data by environment.</a></li>
+      </ul>
+   </div>
 3. Restart the Docker containers.
 4. [Explore the performance observability of your services in Datadog][6].
 
-   <div class="alert alert-info">You can optionally set an environment for your services and other telemetry that pass through the Agent. Read <a href="#env-docker">tagging observability data by environment</a> to learn how. </div>
+### Specifying tracing library versions {#lib-docker}
```
Comment (jonbodner) [inline content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md:117]: These instructions are incorrect. Apologies, I didn't specify this in my initial communication with you.

To specify tracing libraries and versions for docker, set `DD_APM_INSTRUMENTATION_LIBRARIES` when running the  `bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_docker_injection.sh)"` installation script in the step labeled "Install the library injector"

```
DD_APM_INSTRUMENTATION_LIBRARIES="java:1.25.0,python" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_docker_injection.sh)"
```

The rest of the description is correct.

**Review by jonbodner — CHANGES_REQUESTED** [review]
The instructions for specifying language and version for docker were incorrect; I left a comment describing the problem and resolution.

File: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md:117]: @jonbodner - Thanks for the keen eye! I updated this section to correctly set the library versions inside the installation command. Let me know how this looks.

**Review by jonbodner — APPROVED** [review]
LGTM

Thanks!

**Pull request merged**